packages/teams-js/rollup.config.mjs

```diff
@@ -22,12 +22,6 @@ export default [
       entryFileNames: '[name].js',
       sourcemap: false,
       plugins: [terser()],
-      globals: {
-        buffer: 'Buffer',
-        tty: 'tty',
-        util: 'util',
-        os: 'os',
-      },
```

Comment:
Maybe this will come in the pending PR description but I'd love to learn more about the motivation and effect for the changes in this file.

Reply:
See PR description.

TLDR: It's dead code not relevant anymore so I'm removing it for cleanliness.